BUILD.bazel

```diff
@@ -685,6 +685,17 @@ cc_binary(
     ],
 )
 
+cc_binary(
+    name = "job_state_accessor_test",
+    testonly = 1,
+    srcs = ["src/ray/gcs/job_state_accessor_test.cc"],
+    copts = COPTS,
+    deps = [
+        ":gcs",
+        "@com_google_googletest//:gtest_main",
+    ],
+)
```

Comment:
As discussed before, we should change this to a `cc_test`.  But we can do that in the next PR.